src/globals/js/watch.js

```diff
@@ -0,0 +1,66 @@
+import initCheckbox from '../../components/checkbox/checkbox';
+import { componentClasses } from '../../index';
+
+const forEach = Array.prototype.forEach;
```

Comment:
Elsewhere in the library, we generally use the spread operatior plus the array syntax as shorthand. Why swap to calling out the Array prototype forEach here?